app/models/invenio_rdm_record_converter.rb

```diff
@@ -508,4 +517,18 @@ def funding(file_id)
     # remove nil values
     funding_sources.compact
   end
+
+  def map_collections_to_communities
+    @dh_collections.map do |collection_path|
+      prism_community_id_from_collection_path(collection_path)
+    end
+  end
+
+  def prism_community_id_from_collection_path(collection_path)
+    collection_path.each do |collection_path_entry|
+      if community = @@collections_to_communities[collection_path_entry[:id]]
```

Comment:
cute (I am assuming it's a common ruby idiom?)

(minor:
would this be clearer?
```ruby
community = @@collections_to_communities[collection_path_entry[:id]]
return community if community
```
)

Reply:
I'm not sure if it's something people still use, but I've seen it used quite a bit over the years. A lot of the older ruby idioms have fallen out of favor for one reason or another. Usually it's because something could be done a different way that is considered clearer (like you've done here).